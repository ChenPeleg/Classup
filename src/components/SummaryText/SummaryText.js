import React from "react";

const SummaryText = (props) => {
  return (
    <>
      <div style={{ textAlign: "center" }}>
        <span style={{ fontWeight: "Bold" }}> Very Good! </span>
        <br></br>
        <br></br> You've completed this lesson. This is your lesson summary by
        number of attempts to answer each question.
        {props.children}
        <br></br>
        You can practice this lesson again by Pressing "Strat Again". all Of
        your progress will be deleted.
      </div>
    </>
  );
};
export default SummaryText;
