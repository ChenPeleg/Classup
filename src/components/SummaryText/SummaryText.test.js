import React from "react";
import { shallow } from "enzyme";
import SummaryText from "./SummaryText";
const wrap = shallow(<SummaryText />)

describe("<SummaryText>", () => {
  it("renders correctly", () => { expect(wrap).toMatchSnapshot() })
})