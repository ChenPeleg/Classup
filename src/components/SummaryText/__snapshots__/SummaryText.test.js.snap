// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SummaryText> renders correctly 1`] = `ShallowWrapper {}`;
